src/Listener/Symfony/MessengerListener.php

```diff
@@ -28,7 +28,7 @@ public function onMessageReceived(WorkerMessageReceivedEvent $event): void
             if (is_object($message) && !in_array($class = get_class($message), $this->config->getMessageToAssert(), true)) {
                 $this->logger->log(
                     $this->logLevel,
-                    'Untracked message has been detected, add it in your configuration to ensure ZDD compliance.',
+                    "Untracked $class has been detected, add it in your configuration to ensure ZDD compliance.",
```

Comment:
Use [PSR-3 placeholders](https://www.php-fig.org/psr/psr-3/#12-message) instead. Like this you have best of both world and it will be up the user of the bundle to choose what behaviors he/she wants.

```suggestion
                    "Untracked {message} has been detected, add it in your configuration to ensure ZDD compliance.",
```